src/ParseMonad.hs

```diff
@@ -64,7 +64,7 @@ alexGetByte (p,_,[],(c:s))  = let p' = alexMove p c
 -- assuming the usual eight character tab stops.
 
 data AlexPosn = AlexPn !Int !Int !Int
-        deriving (Eq,Show)
+        deriving (Eq, Show, Ord)
```

Comment:
Sorry, now that I have another look, I see that you only add this to `alex` itself not to the _lexers_ that `alex` generates.

What is the point of this?

Reply:
ah, well-spotted! fixing this :)